hack/tests/e2e-ansible.sh

```diff
@@ -18,22 +18,14 @@ setup_envs $tmp_sdk_root
 
 deploy_operator() {
     header_text "Running deploy operator"
-    kubectl create -f "$OPERATORDIR/deploy/service_account.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/role.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/role_binding.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/crds/ansible.example.com_memcacheds_crd.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/crds/ansible.example.com_foos_crd.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/operator.yaml"
+    kustomize build --load_restrictor none config/testing | kubectl apply -f -
+    kubectl create clusterrolebinding memcached-operator-system-metrics-reader --clusterrole=osdk-metrics-reader --serviceaccount=default:default
```

Comment:
We should use the makefile targets to ensure that is working. See the helm test for the new layout in shell: https://github.com/operator-framework/operator-sdk/blob/c1b615c6f01d303ece5bcb06f46565ae0b6b7999/hack/tests/e2e-helm.sh#L22-L27

Reply:
@fabianvf ran into a feature of kustomize that made it impossible to reuse the patches in the testing overlays. https://github.com/kubernetes-sigs/kustomize/issues/937

So for these tests, we need to run kustomize outside of the makefile to add `--load_restrictor none`

Reply:
IMO the test should ensure that the users are able to create an Ansible operator by following the happy path with the makefile targets (if it is not possible so it means that we have issues to be fixed) which are:

- mkdir <name>
- cd <name>
- operator-sdk init --plugins=ansible ... (init project)
- operator-sdk create api ... (create api)
- make install
- make docker-build
- make deploy 
- make uninstall
- make undeploy

Also, the test should ensure that the metrics are exported and that is possible reconcile the CR in another namespace that is not the same used by the operator. E.g https://github.com/operator-framework/operator-sdk/blob/c1b615c6f01d303ece5bcb06f46565ae0b6b7999/hack/tests/e2e-helm.sh